app/models/blueprint.rb

```diff
@@ -52,6 +52,8 @@ def copy_service_template(blueprint, service_template, is_top = false, new_attri
       duplicate_resource_actions(service_template, new_template, is_top)
       duplicate_service_resources(blueprint, service_template, new_template)
       update_copied_service_template(blueprint, new_template, new_attributes)
+      duplicate_custom_buttons(service_template, new_template)
+      duplicate_custom_button_sets(service_template, new_template)
```

Comment:
@bdunne Yes. I was thinking the `new_attributes` that the caller might pass in. What if it contains new custom_buttons to overwrite the copied buttons?